composer/cicd_sample/dags/example2_dag_test.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from airflow import models
+import airflow.utils.db
+import internal_unit_testing
+import pytest
+
+
+@pytest.fixture(autouse=True, scope="module")
+def initalize_airflow_database():
```

Comment:
```suggestion
def initialize_airflow_database():
```

Reply:
This is actually dated and unnecessary - great catch. Removing altogether.